example/requirements.txt

```diff
@@ -1 +1,2 @@
+fire
```

Comment:
[Out of this PR scope] 

I think now we should use the Poetry and control the outside library's version.

I made #28